docs/content/maintenance/dedicated-compaction.md

```diff
@@ -107,6 +107,7 @@ Run the following command to submit a compaction job for the table.
     --database <database-name> \ 
     --table <table-name> \
     [--partition <partition-name>] \
+    [--compact_strategy <minor / full>] \
```

Comment:
Now in paimon, there are already three compact_strategy, FullCompaction, UniversalCompaction, ForceUpLevel0Compaction.
I think here we change compact_strategy to compact_type may better.?

Reply:
Essentially there are only two strategies `full` or `minor`, so using `compact_strategy ` is more suitable for this scenario.